.gitlab-ci.yml

```diff
@@ -1,6 +1,48 @@
-test:
+variables:
+  CI_IMAGE: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/integrations-core:latest
+  TAGGER_EMAIL: packages@datadoghq.com
+  TAGGER_NAME: ci.integrations-core
+
+release-auto:
+  image: $CI_IMAGE
   only:
-    # e.g.: any_check-X.Y.Z-rc.N, zk-1.15.0
+    - master
+  script:
+    - ddev --version
+    - ddev config set core .
+    # Prefix every line with a timestamp
+    - ./.gitlab/tag-release.sh 2>&1 | ts "[%H:%M:%S %Z]  "
+  tags: [ "runner:main", "size:large" ]
+
+release-manual:
+  image: $CI_IMAGE
+  only:
+    # Integration release tags e.g. any_check-X.Y.Z-rc.N
     - /.*-\d+\.\d+\.\d+(-(rc|pre|alpha|beta)\.\d+)?$/
-  script: if [ -n "$DD_TEST_CMD" ] ; then eval "$DD_TEST_CMD" ; else echo "skipping, DD_TEST_CMD is not set" ; fi
+  script:
+    # Get tagger info
+    - tagger=$(git show $CI_COMMIT_TAG | head -2 | tail -1)
+    # The automatic release builder will trigger this job as a side-effect of
+    # tagging releases. To prevent multiple redundant builds we don't trigger
+    # the pipeline unless the tag was applied manually.
+    - |
+      if [[ "$tagger" =~ "Tagger: $TAGGER_NAME <$TAGGER_EMAIL>" ]]; then
+          echo "Skipping, packages have already been built"
+      else
+          ./.gitlab/build-packages.sh
+      fi
   tags: [ "runner:main", "size:large" ]
+
+release-image-builder:
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/docker:18.03.1
+  only:
+    changes:
+      - datadog_checks_dev/**
```

Comment:
Why not make this manual instead of on every change? You don't make a release on every change (and since you're using pipy it will just build the same thing over again

Reply:
It uses master actually, not PyPI

Reply:
Oh I see, misread that

Reply:
Why do you want to use this instead of pypi?

Reply:
Mostly so any breakages don't get shipped to PyPI and are caught right away. In future we could select only the 2 relevant files for changes (after we refactor our massive release.py into separate files)

Reply:
Wait are you suggesting we test your changes in production?

Reply:
Are there other consumers of this function we should worry about other than our own release process?

Reply:
Nope we're the only ones that use this for now

Reply:
I am not certain that it's a great idea to test this in production, but I will approve anyway